apis/r/src/rutilities.cpp

```diff
@@ -1,146 +1,211 @@
 
-#include <Rcpp.h>               // for R interface to C++
-#include <nanoarrow.h>          // for C interface to Arrow
+#include <Rcpp.h>       // for R interface to C++
+#include <nanoarrow.h>  // for C interface to Arrow
 
 // We get these via nanoarrow and must cannot include carrow.h again
 #define ARROW_SCHEMA_AND_ARRAY_DEFINED 1
 #include <tiledbsoma/tiledbsoma>
 
-#include "rutilities.h"         // local declarations
+#include "rutilities.h"  // local declarations
 
 namespace tdbs = tiledbsoma;
 
-void apply_dim_points(tdbs::SOMAArrayReader *sr,
-                      std::unordered_map<std::string, std::shared_ptr<tiledb::Dimension>>& name2dim,
-                      Rcpp::List lst) {
+void apply_dim_points(
+    tdbs::SOMAArray* sr,
```

Comment:
was this `clang-format` -- or something else? just curious why these are appearing